Proteomics/ProteolyticDigestion/ProteolyticPeptide.cs

```diff
@@ -314,7 +314,14 @@ private bool CanBeCTerminalMod(Modification variableModification, int peptideLen
                 {
                     case "N-terminal.":
                     case "Peptide N-terminal.":
-                        if (ModificationLocalization.ModFits(mod, Protein.BaseSequence, 1, peptideLength, OneBasedStartResidueInProtein))
+                        if (mod.ModificationType == "Protease" && ModificationLocalization.ModFits(mod, Protein.BaseSequence, 1, peptideLength, OneBasedStartResidueInProtein))
```

Comment:
is the purpose of this section to put the mod on peptide N-term while avoiding protein N-Term?

Reply:
Yes. same for C-term.